mmocr/apis/inferencers/mmocr_inferencer.py

```diff
@@ -105,34 +105,51 @@ def _inputs2ndarrray(self, inputs: List[InputsType]) -> List[np.ndarray]:
                                           'supported yet.')
         return new_inputs
 
-    def forward(self, inputs: InputsType, batch_size: int,
+    def forward(self,
+                inputs: InputsType,
+                batch_size: int = 1,
+                det_batch_size: int = None,
+                rec_batch_size: int = None,
+                kie_batch_size: int = None,
```

Comment:
```suggestion
                det_batch_size: Optional[int] = None,
                rec_batch_size: Optional[int] = None,
                kie_batch_size: Optional[int] = None,
```

Reply:
Thanks for the feedback! 

I noticed that all optional input parameters in the **\_\_call__()** and **forward()** functions do not have the **Optional** keyword specified. Wanna clarify if you would suggest me adding **Optional** to **det_batch_size**, **rec_batch_size**, and **kie_batch_size** only, or adding **Optional** to all optional parameters in these two functions. Similar questions to the doc string.